site/app/templates/officeHoursQueue/CurrentQueue.twig

```diff
@@ -24,7 +24,7 @@
 
           {% if entry['current_state'] == 'waiting' %}
             <td>
-              <form method="post" action="{{base_url}}/{{entry['queue_code'] | url_encode }}/startHelp" {% if entry['paused'] %}onsubmit="return confirm('Are you sure you want to start helping this student? This student is currently paused.');"{% endif %}>
+             <form method="post" action="{{base_url}}/{{entry['queue_code'] | url_encode }}/startHelp" {% if entry['paused'] %}onsubmit="return confirm('Are you sure you want to start helping this student? This student is currently paused.');"{% endif %}> 
```

Comment:
This change seems unrelated and should be reverted.

Reply:
Ah yes, I changed this when messing around and trying to do the 3 variable solution and forgot to change it back when I removed it.